Sources/DVR/Session.swift

```diff
@@ -141,7 +160,7 @@ open class Session: URLSession {
         return Cassette(dictionary: json)
     }
 
-    func finishTask(_ task: URLSessionTask, interaction: Interaction, playback: Bool) {
+    func finishTaskAndPersist(_ task: URLSessionTask, interaction: Interaction, playback: Bool) {
         needsPersistence = (needsPersistence || !playback)
```

Comment:
This is interesting. If the method says "Do persist" why do we have this check? If it doesn't need persistence, should we not be calling the other method?

Reply:
I see what you're getting at. It no longer needs that logic. But I think the names should be adjusted because they're not clear.